src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -693,7 +693,14 @@ int goto_instrument_parse_optionst::doit()
       goto_inline(goto_model, get_message_handler());
 
       status() << "Accelerating" << eom;
-      accelerate_functions(goto_model, cmdline.isset("z3"));
+      satcheck_infot satcheck_info;
+      if(cmdline.isset("random-seed"))
+        satcheck_info.random_seed=std::stod(cmdline.get_value("random-seed"));
+      if(cmdline.isset("random-seed"))
+        satcheck_info.random_var_freq
+          =std::stod(cmdline.get_value("random-var-freq"));
+      accelerate_functions(
+        goto_model, cmdline.isset("z3"), satcheck_info);
```

Comment:
The command-line options are never added to `goto-instrument`, are they? It seems they just exist in `cbmc`.